source/documentation/internal/low-github-actions-minutes-procedure.html.md.erb

```diff
@@ -0,0 +1,26 @@
+---
+owner_slack: "#operations-engineering-alerts"
+title: Low GitHub Actions Minutes Procedure
+last_reviewed_on: 2024-03-19
+review_in: 3 months
+---
+
+# Low GitHub Actions Minutes Procedure
+
+## Overview
+
+This document outlines the steps to follow when an alarm is triggered in our Slack channel, indicating potential issues with low GitHub Actions minutes.
+
+## Steps to Follow
+
+### Communicate with Customers:
```

Comment:
Is it worth including a template for comms? Here is an example - https://mojdt.slack.com/archives/C02D2NEF9CJ/p1693299313645569

If we used a template it would make this step easier to whoever is doing the comms, plus adds consistency to our messages?